aten/src/ATen/core/Tensor.h

```diff
@@ -566,6 +567,10 @@ class CAFFE2_API Tensor {
   std::vector<Tensor> unbind(int64_t dim=0) const;
   Tensor to_sparse(int64_t sparse_dim) const;
   Tensor to_sparse() const;
+  Tensor quantize_linear(double scale, int64_t zero_point) const;
+  Tensor dequantize() const;
+  Scalar q_scale() const;
+  Scalar q_zero_point() const;
```

Comment:
Can we have quantize_linear take qparams as an input where qparams is a struct holding scale and zero_point? This way the top level quantize and dequantize apis will not depend on the details of the quantizer.

Reply:
Yeah this will be next pr after we close on the discussion points.